README.md

```diff
@@ -161,6 +161,21 @@ url: https://example.com/
 ignore_connection_errors: true
 ```
 
+Similarly, you might want to ignore some (temporary) HTTP errors on the
+server side:
+
+```yaml
+url: https://example.com/
+ignore_http_error_codes: 503
```

Comment:
Some other temporary error codes that you could include in this example:

408, 429, 500, 502, 503, 504